src/main/java/fredbot/FredBot.java

```diff
@@ -0,0 +1,301 @@
+import java.util.Scanner;
+
+public class FredBot {
+    private static ArrayList<Task> allTasks;
+    private static final int TASK_CAPACITY = 100;
+    private static int count;
+
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_DELETE = "delete";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_UNMARK = "unmark";
+
+    private static final String MESSAGE_ADD = "Are you sure you'll ever get to it? Fine, I've added this task:";
+    private static final String MESSAGE_DELETE = "Finally lifting my load? I've removed this task:";
+    private static final String MESSAGE_EMPTY_LIST = "Go and touch some grass... your list is empty.";
+    private static final String MESSAGE_EMPTY_DESCRIPTION = "I can't do that if you don't give me the description...";
+    private static final String MESSAGE_ERROR = "What did you do... Something went wrong.";
+    private static final String MESSAGE_UNKNOWN_COMMAND = "I have no idea what you just said.";
+    private static final String MESSAGE_GOODBYE = "Finally... goodbye.";
+    private static final String MESSAGE_LIST_HEADER = "Mr Busy over here has these tasks in his list:";
+    private static final String MESSAGE_MARK = "Ok and do you want a medal for that? I've marked this as done:";
+    private static final String MESSAGE_UNMARK = "Why am I not surprised... I've marked this task as not done yet:";
+    private static final String MESSAGE_WELCOME = "Yo I'm FredBot. What do you want with me?";
+
+    private static final String PREFIX_BY = "/by";
+    private static final String PREFIX_FROM = "/from";
+    private static final String PREFIX_TO = "/to";
+
+    private static final String FILE_PATH = "data/fredbot.txt";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        loadFredBot();
+        showWelcomeMessage();
+        while (true) {
+            String userInput = readUserInput();
+            executeCommand(userInput);
+            saveFredBot();
+        }
+    }
+
+    private static void loadFredBot() {
+        initTaskList();
+        File f = new File(FILE_PATH);
+        try {
+            loadTaskList(f);
+        } catch (FileNotFoundException e) {
+            initSaveFile(f);
+        }
+    }
+
+    private static void loadTaskList(File f) throws FileNotFoundException {
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String task = s.nextLine();
+            try {
+                processLine(task);
+            } catch (EmptyDescriptionException e) {
+                System.out.println(MESSAGE_EMPTY_DESCRIPTION);
+            }
+        }
+    }
```

Comment:
You could opt to use more descriptive variable names for the variables `f` and `s`.